src/extractor.coffee

```diff
@@ -106,6 +106,34 @@ module.exports =
 
     null
 
+  # Find any links in the doc
+  links: (doc, topNode, lang) ->
+    gravityItems = []
+    links = []
+    removeGravityItems = (gravityItems) ->
```

Comment:
There were no side effects later (all tests were the same in either direction). I don't remember why I had added this loop before. After removing and running the tests again, the same results are returned without it. So let's leave it out.